### DISCUSSION TITLE
Looking for uniqueness in portal accounts

### DISCUSSION THREAD

**Discussion by mikeegan400** [microsoft/MIMWAL#110] [General]
We have a group of users that created in the MIM portal and, then, the account is created in AD.  I am looking for a way to compare the new users first name and last name to the other portal accounts.  I have been looking for a solution for the last 3 weeks and I can't figure out how to do this with the MIMWAL activities.

Thank you,
Mike

**Comment by FredrikMelby** [comment]
Look at the "Verify Request Activity" that you can use in the authorization phase of MIM Request Processing:

Use the verify the resource as unique option and write x-path filter to find conflicting user accounts (if this is what you want?):
```
/Person[FirstName = '[//Delta/FirstName]']
/Person[LastName= '[//Delta/LastName]']
```

MIMWAL will then search for existing Person objects with the same firstname/lastname set and you can also write a custom "denied" message to show when conflicting names are typed in..
